.github/workflows/ci.yml

```diff
@@ -0,0 +1,70 @@
+name: Continuous Integration
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  htmlproofer:
+    name: HTMLProofer
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout EIP Repository
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+      
+      - name: Install OpenSSL
+        run: sudo apt-get update && sudo apt-get install -y libcurl4-openssl-dev
+    
+      - name: Install Ruby
+        uses: ruby/setup-ruby@08245253a76fa4d1e459b7809579c62bd9eb718a
+        with:
+          ruby-version: '2.6.0'
+          bundler-cache: true
+        
+      - name: Build Website
+        run: |
+          bundle exec jekyll doctor
+          bundle exec jekyll build
+      
+      - name: HTML Proofer
+        run: bundle exec htmlproofer $HTMLPROOFER_OPTIONS --disable-external
+        env:
+          HTMLPROOFER_OPTIONS: ./_site --internal-domains=eips.ethereum.org --check-html --check-opengraph --report-missing-names --log-level=:debug --assume-extension --empty-alt-ignore --timeframe=6w --url-ignore=/EIPS/eip-1,EIPS/eip-1,/EIPS/eip-107,/EIPS/eip-858
```

Comment:
Feels like we could inline this rather than having it as an environment variable.  I don't think we gain much at all by splitting it out, especially since *some* options are left out (`--disable-external`).